cmd/nitro-val/nitro_val.go

```diff
@@ -70,7 +70,6 @@ func mainImpl() int {
 	nodeConfig.WS.Apply(&stackConf)
 	nodeConfig.Auth.Apply(&stackConf)
 	nodeConfig.IPC.Apply(&stackConf)
-	nodeConfig.P2P.Apply(&stackConf)
```

Comment:
let's remove this function as well?

Reply:
Removed along with the whole P2PConfig struct. We can revert this PR when we're ready to reintroduce P2P.